src/tables/components/table-row.js

```diff
@@ -14,8 +14,8 @@ const propTypes = {
 }
 
 const DefaultRowComponent = ({ children }) => <tr>{ children }</tr> // eslint-disable-line
-const DefaultCellComponent = ({ className, onClick, placeholder, value }) => // eslint-disable-line
-  <td { ...{ className, onClick }}>{ isNil(value) ? placeholder : value }</td>
+const DefaultCellComponent = ({ className, onClick, placeholder, value, ...rest }) => // eslint-disable-line
+  <td { ...{ className, onClick, ...rest }}>{ isNil(value) ? placeholder : value }</td>
```

Comment:
We should leverage the `filterInvalidDOMProps` util here. Otherwise, we'll see warnings like this:
![Image 2021-11-11 at 8 49 43 AM](https://user-images.githubusercontent.com/20426095/141309353-10e19074-8b31-41f4-828f-762ebe3e3274.jpg)